core/templates/pages/skill-editor-page/editor-tab/skill-editor-main-tab.directive.ts

```diff
@@ -114,6 +116,9 @@ angular.module('oppia').directive('skillEditorMainTab', [
             $scope.topicName = null;
             $scope.subtopicName = null;
             PageTitleService.setPageTitleForMobileView('Skill Editor');
+            $timeout(function() {
+              FocusManagerService.setFocus('newQuestionBtn');
```

Comment:
Does this always ensure that the button is focused i.e. setting a 0 timeout?

Reply:
There can never be a timeout of 0 seconds actually, it gives it the most least possible value (I read it to be  around 4 ms in the documentation) and fires up after all the tasks in the script have completed.As there are a very few asynchronous functions on the page it works with this least value.

Reply:
That doesn't seem very deterministic. i.e. it could be possible that the async stuff hasn't completed on time (e.g. in slow connections),

Reply:
Hm, increasing the timeout duration is not quite the fix I had in mind. Can you tell me what exactly is the condition for the input "newQuestionBtn" to show on the screen?